pkg/authenticator/jwt/tests/authenticator_test_server.go

```diff
@@ -0,0 +1,48 @@
+package tests
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+)
+
+type TestAuthServer struct {
```

Comment:
This file seems to be a slimmed down version of `pkg/authenticator/k8s/tests/authenticator_test_server.go`. Can we move that a location where it can be reused instead of duplicated?

Reply:
Good point thanks